### DISCUSSION TITLE
Understanding Negation - Challenge

### DISCUSSION THREAD

**Discussion by eduardoblanco** [eduardoblanco/csc696H-Spring24#4] [General]
I am going to post some sort of challenge for each topic. Nothing too difficult or intellectually challenging. I just think it is important to get your hands dirty.

1. Here is the [repo](https://github.com/mosharafhossain/large-afin-and-nlu) for the _Leveraging Affirmative Interpretations [...]_ paper. Who can find a pair in the [dataset](https://raw.githubusercontent.com/mosharafhossain/large-afin-and-nlu/main/affirmative-interpretation-generation/data/large-afin/large-afin.jsonl) (150k pairs) that does not meet the definitions?

1. Here is the [repo](https://github.com/AbhilashaRavichander/CondaQA) for CondaQA. Who can find a question-answer pair that does not follow the description in the paper?

**Comment by mayanks43** [comment]
1. One example: {"sentence": "Many of the opponents of Euro Communism decided that they had to create their own communist party that was not , as they saw it , revisionist .", "affirmative_interpretation": "Many of the anti - Eurocommunists decided that they needed to form their own anti - revisionist Communist party ."}. There's a negative affix "anti-" in the affirmative interpretation.

**Reply by eduardoblanco** [reply]
Good one. I think that it has to do with the collection of negation cues. The detector we used probably only identified "not." On a related not, I don;t know how to write "anti - revisionist" without a negation. Is there a word for that?

@brianrez had some ideas (never really explored) on figuring out when it is worth attempting to generate an affirmative interpretation.

**Reply by mayanks43** [reply]
> On a related not, I don;t know how to write "anti - revisionist" without a negation. Is there a word for that?

TBH, I don't think an affirmative interpretation exists for every sentence with a negation.

**Reply by shivammathur33** [reply]
pro-traditionalist?

**Comment by devsawgrad** [comment]
1. {"sentence": "Some fiction may include non - fiction elements .", "affirmative_interpretation": "Some fiction may include nonfictional elements ."} is also one more example. no change in the interpretation basically.
